### PR TITLE
OSSM-2378: Use Maistra proxy CentOS Stream 8 img

### PR DESCRIPTION
**Pull request by bartoszmajsak**
For integration tests. Based on (#609)

* Build container images with docker

Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

* Use maistra proxy and CentOS Stream 8 as a base image for proxyv2

Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

* Test VMs using CentOS Stream 8

Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

* Temporarily skip failing security tests

Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

* Temporarily skip tests suite telemetry.prometheus.wasm

Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

Signed-off-by: Jacek Ewertowski <jewertow@redhat.com>

**Please provide a description of this PR:**

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: tests/integration/security/file_mounted_certs/p2p_mtls_test.go [hunk 1]
```diff
@@ -39,6 +39,8 @@ const (
 )
 
 func TestClientToServiceTls(t *testing.T) {
+	// FIXME: https://issues.redhat.com/browse/OSSM-755
+	t.Skip("https://github.com/istio/istio/issues/0")
```
Comment (jwendell) [inline tests/integration/security/file_mounted_certs/p2p_mtls_test.go]: I think this is fixed so we don't need to skip it?

File: tests/integration/security/filebased_tls_origination/destination_rule_tls_test.go [hunk 2]
```diff
@@ -29,6 +29,8 @@ import (
 // TestDestinationRuleTls tests that MUTUAL tls mode is respected in DestinationRule.
 // This sets up a client and server with appropriate cert config and ensures we can successfully send a message.
 func TestDestinationRuleTls(t *testing.T) {
+	// FIXME: https://issues.redhat.com/browse/OSSM-755
+	t.Skip("https://github.com/istio/istio/issues/0")
```
Comment (jwendell) [inline tests/integration/security/filebased_tls_origination/destination_rule_tls_test.go]: ditto

File: tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go [hunk 3]
```diff
@@ -43,6 +43,8 @@ import (
 // This test brings up an egress gateway to originate TLS connection. The test will ensure that requests
 // are routed securely through the egress gateway and that the TLS origination happens at the gateway.
 func TestEgressGatewayTls(t *testing.T) {
+	// FIXME: https://issues.redhat.com/browse/OSSM-755
+	t.Skip("https://github.com/istio/istio/issues/0")
```
Comment (jwendell) [inline tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go]: ditto

**Review by jwendell — APPROVED** [review]

**Comment by bartoszmajsak** [comment]
Thanks @jwendell

**Pull request merged**